Makefile

```diff
@@ -15,7 +15,8 @@ cli.prepare:
 
 .PHONY: proto.generate
 proto.generate:
-	cd proto && buf generate
+	cd proto && buf generate --exclude-path rill/ui
 	cd proto && buf generate --template buf.gen.openapi-admin.yaml --path rill/admin
 	cd proto && buf generate --template buf.gen.openapi-runtime.yaml --path rill/runtime
+	cd proto && buf generate --template buf.gen.ui.yaml --output ../web-common/src/proto
```

Comment:
It doesn't support specifying the `--output` as `out: ../web-common/src/proto/gen` in the `buf.gen.ui.yaml`? 

I'm pretty unhappy about the need for CLI args on `buf` in general. Why don't they allow putting all the config in the YAML?

Reply:
They actually do support it. Have updated the config